bims/static/js/views/olmap.js

```diff
@@ -503,6 +504,11 @@ define([
             if(!hideBimsInfo && bimsInfoContent) {
                 $('#general-info-modal').fadeIn()
             }
+        },
+        resetMap: function () {
+            var center = [22.937506, -30.559482];
```

Comment:
you need to use https://github.com/kartoza/django-bims/blob/85fbc5bbe5b74875045ad079de3fcd9ec2d1f0c6/bims/static/js/views/olmap.js#L242

maybe safe itu as "initCenter"